Remove `generated-objects-labels` parameter (#880)
kyma-project/api-gateway: docs/release-notes/2.2.0.md

- Introduce PodDisruptionBudget for Ory Oathkeeper. For more information, see the [PR](https://github.com/kyma-project/api-gateway/pull/853).
- Extend APIRule access strategies with `no_auth` to restrict accessible methods, thereby minimizing the exposure of the API.
- Introduce consistent labeling for the Kyma modules. For more information, see the [documentation](https://kyma-project.io/#/api-gateway/user/00-10-overview-api-gateway-controller?id=labeling-resources).

> - Removed `--generated-objects-labels` parameter and all the code that was used by it.

Reviewer: ```suggestion
- Remove the **--generated-objects-labels** parameter and all the code that it used.
```
